tests/unit/nc-button.spec.js

```diff
@@ -0,0 +1,36 @@
+import { mount } from '@vue/test-utils'
+import ncButton from '@/components/nc-button.vue'
+
+describe('ncButton', () => {
+  const defaultProps = {
+    className: undefined,
+    disabled: false
+  }
+  describe('When is mounted without props', () => {
+    let wrapper
+    let result
+    beforeAll(() => {
+      wrapper = mount(ncButton)
+      result = wrapper.props()
+    })
+    it('should set all its default props', () => {
```

Comment:
Añadiría saltos de linea para ayudar a la legibilidad (esto ya es opinion personal)
```suggestion

    it('should set all its default props', () => {
```

Reply:
Añadida linea